facilitator/src/bin/facilitator.rs

```diff
@@ -466,7 +466,20 @@ fn main() -> Result<(), anyhow::Error> {
                             "Base64 encoded X9.62 uncompressed public key for the PHA \
                             server",
                         )
-                        .required(true),
+                )
+                .arg(
+                    Arg::with_name("pha-manifest-url")
```

Comment:
From reading the remainder of the changes, it seems like this is the full URL to a specific manifest, e.g., "https://somebase.url/ta-ta-ingestor-1-manifest.json`. Could you instead take parameters for ingestor name and locality and then use the existing `SpecificManifest::from_https`?